eng/tools/azure-sdk-tools/azpysdk/Check.py

```diff
@@ -0,0 +1,31 @@
+import abc
+import argparse
+from typing import Sequence, Optional, List, Any
+
+class Check(abc.ABC):
```

Comment:
Can we add a readme eventually here on how to trigger the checks if we dont already have one

Reply:
Yeah definitely.